guides/common/modules/ref_webhooks-templates.adoc

```diff
@@ -0,0 +1,12 @@
+[id="webhooks-templates_{context}"]
+= Templates
+
```

Comment:
The text in this section is without any developed context, and reads like a group of random statements. 
As a user, why am I here reading this? 
Some structure would be beneficial.